stock_warehouse_calendar_orderpoint/models/res_company.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    orderpoint_calendar_id = fields.Many2one(
```

Comment:
It is only used as default value in `stock.warehouse`, so it's a bit confusing to have it in general settings.

Imagine you have already some warehouses configured (which you do in all databases when you install this module), and then you change this setting.

The warehouses' calendars wouldn't be changed
see (`StockWarehouse._default_orderpoint_calendar_id`)


IMO you should drop this field on the company / general settings. Or at least rename it to `default_orderpoint_*` and clearly state that's going to be used as a default for new warehouses... but TBH having a setting to set a default value on a warehouse is overkill. You only create warehouses once..

Reply:
Yes it was in discussion, we are waiting feedback after some usage by the customer so we'll probably rework this wobbly part.